final/server/dist/src/datasources/track-api.js

```diff
@@ -0,0 +1,26 @@
+import { RESTDataSource } from '@apollo/datasource-rest';
+export class TrackAPI extends RESTDataSource {
```

Comment:
do we even need to generate a `dist` folder? Running the app converts it on the fly it seems?